app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError
+                        )
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("A visitor should be vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("A visitor should be vaccinated")
```

Comment:
write more informative message for this error, f.e. "A visitor should have up-to-date vaccine"

Reply:
thanks for this feedback..  It will be revised